src/main.rs

```diff
@@ -30,6 +30,7 @@ fn main() {
     opts.optflag("y", "yes", "skip the check for deleting branches");
     opts.optopt("R", "remote", "changes the git remote used (default is origin)", "REMOTE");
     opts.optopt("b", "branch", "changes the base for merged branches (default is master)", "BRANCH");
+    opts.optopt("i", "ignore", "ignore given branch", "BRANCH");
```

Comment:
I imagine that we'll want to be able to pass multiple `-i` flags to ignore multiple branches, we should make this an `optmulti`.

https://doc.rust-lang.org/getopts/getopts/struct.Options.html#method.optmulti
https://doc.rust-lang.org/getopts/getopts/struct.Matches.html#method.opt_strs